sdkproject/Assets/Mapbox/Unity/Editor/GeocodeAttributeSearchWindow.cs

```diff
@@ -121,6 +124,10 @@ void OnGUI()
 							_coordinateProperty.serializedObject.ApplyModifiedProperties();
 							EditorUtility.SetDirty(_coordinateProperty.serializedObject.targetObject);
 
+							if (_abstractMap != null)
```

Comment:
I am checking to see if the AbstractMap instance passed via open is null so that update events can be optional; two options here. If we always want any instance of the GeocodeAttributeSearchWindow to send an update event to abstract map, then I'll remove the null check and Open() passed instance variable. If we do not, then I will replace the passed instance variable with an optional bool to preserve the option of updating.